titus-server-master/src/main/java/com/netflix/titus/master/scheduler/opportunistic/OpportunisticCpuAvailability.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.titus.master.scheduler.opportunistic;
+
+import java.time.Instant;
+
+import com.netflix.titus.common.util.time.Clock;
+
+public class OpportunisticCpuAvailability {
+    private final String allocationId;
+    private final Instant expiresAt;
+    private final int count;
```

Comment:
Could we make this a `float` instead? so it matches https://github.com/Netflix-Skunkworks/titus-isolate/pull/198/files#diff-88d117d04068fc260a43286f0d058b59R7

Reply:
I would really, *really*, prefer to avoid it and avoid headaches with floating point based math everywhere

Reply:
We can revisit if we ever add fractional CPUs (which can also be done with ints, e.g.: `1.5 CPU == 150, 2.3 CPU == 230, etc`